documentation/components/JsonSchema.rst

```diff
@@ -148,6 +148,7 @@ Other options are available to customize the generated code:
    that forces them to be present during denormalization. By default it is disabled.
  * ``validation``: Will enable validation following JSON Schema validation specification. By default it is disabled. You
    can read more about it on the dedicated guide: :doc:`/guides/validation`.
+ * ``optimize-null-management``: Will enable an optimized way to manage null values. By default it is disabled.
```

Comment:
I'm not sure this name is descriptive enough, what do you think about `force-null-value` ? (Enabled by default)

Reply:
Enabled by default is dangerous, in my case it works well because all the fields in my responses are "required" in the OpenAPI. If they are not required in responses, behavior will be a little different. The best option is to apply this change only for requests to send and not responses but it's not possible

Reply:
I don't mean enabling this feature by default but enabling the parameter, enabled would be current behavior and disabling will be your new feature :wink: 

It's only a mater of how naming things and what make the most sense for this feature, it's only a proposition if you have other idea feel free to give other options !

Reply:
I changed just a little the suggested name to `include-null-value`, tests passed so everything seems good